src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,26 @@
-export const PostInfo = () => <>Put the post here</>;
+import React from 'react';
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+        <UserInfo user={post.user} />
```

Comment:
This element should only be rendered if the user exists. As far as I remember, it was specified somewhere that the user may not be found